web/styles/app_components.css

```diff
@@ -1221,14 +1222,18 @@ div.overlay {
         align-items: center;
         box-sizing: border-box;
         width: 100%;
-        height: 100%;
-        margin: 1px;
+        height: 90%;
+        margin: 1.5px;
         cursor: pointer;
         border-radius: 5px;
 
         &:hover:not(.selected-tab) {
-            background-color: hsl(0deg 0% 100%);
+            background-color: hsl(0deg 0% 100% / 60%);
+        }
+
+        &:active:not(.selected-tab) {
             outline: 1px solid hsl(0deg 0% 0% / 30%);
+            outline-offset: -1px;
```

Comment:
I don't understand why this is an outline; an outline with offset -1 is basically a border, and the `border-box` sizing in this area should mean that setting a humble border here would work without incident.

Reply:
If we use border; then on action (mouse click) we add border which creates a wiggle effect.

Reply:
Ah okay. I'd usually just drop padding by a pixel on a hover/active effect that adds a border, but this works, too.